### PR TITLE
Added Duplicacy configuration files

### PR DESCRIPTION
**Pull request by morpheus65535**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

### CODE REVIEW

**Review by Roxedus — CHANGES_REQUESTED** [review]
The 307 and 301 redirects should be consolidated, i would also see if one could do a condition for the returns, like the plex subfolder example

**Comment by morpheus65535** [comment]
> The 307 and 301 redirects should be consolidated, i would also see if one could do a condition for the returns, like the plex subfolder example

I don't see how it can be done since for Plex all the file reside in a `web` subdirectory where duplicacy store it's files at root. Doind the same as plex would mean that every requests for any location would result in a redirect to duplicacy.

Maybe someone else will come with a hint on how to fix this but I don't see any cleaner than what I did.

**Review by nemchik — CHANGES_REQUESTED** [review]
The proxy templates have been updated with new descriptive comments and auth lines. Please update your PR accordingly.

Ref:
https://github.com/linuxserver/reverse-proxy-confs/blob/master/_template.subdomain.conf.sample
https://github.com/linuxserver/reverse-proxy-confs/blob/master/_template.subfolder.conf.sample

File: duplicacy.subfolder.conf.sample [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    return 307 $scheme://$host/duplicacy/start_stop_schedule;
+}
+
+location /stop_restore {
+    return 307 $scheme://$host/duplicacy/stop_restore;
+}
+
+location /update_listing_session {
+    return 307 $scheme://$host/duplicacy/update_listing_session;
+}
```
Comment (nemchik) [inline duplicacy.subfolder.conf.sample:99]: All these locations are not beneath the subfolder (ex: `location /duplicacy/update_listing_session {`)
Does this app actually work with subfolder?

Comment (morpheus65535) [reply duplicacy.subfolder.conf.sample:99]: Yes but it's not perfect. It require to add each and every path in the proxy config and conflict may happens with other software. Not sure if it's robust enough to be included anymore.

Comment (nemchik) [reply duplicacy.subfolder.conf.sample:99]: Let's just keep the subdomain file for now. If you don't mind, delete the subfolder file and i'll review/accept the subdomain conf.

Comment (morpheus65535) [reply duplicacy.subfolder.conf.sample]: Done!

**Review by nemchik — APPROVED** [review]

**Pull request merged**